unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -1367,9 +1367,10 @@ public class SendVideoProgressEvent
 
         public static void SendSaveAvatar(AvatarModel avatar, Texture2D face256Snapshot, Texture2D bodySnapshot, bool isSignUpFlow = false)
         {
+
             var payload = new SaveAvatarPayload()
             {
-                avatar = avatar,
+                avatar = avatar.ToAvatarModelDto(),
```

Comment:
You could replace the parameter to `AvatarModelDTO` instead of doing an implicit conversion